src/core-layers/point-cloud-layer/point-cloud-layer-vertex-64.glsl.js

```diff
@@ -39,29 +39,23 @@ void main(void) {
   // position on the containing square in [-1, 1] space
   unitPosition = positions.xy;
 
-  vec4 instancePositions64xy = vec4(
-    instancePositions.x, instancePositions64xyLow.x,
-    instancePositions.y, instancePositions64xyLow.y);
+  vec4 position_worldspace;
 
-  vec2 projected_coord_xy[2];
-  project_position_fp64(instancePositions64xy, projected_coord_xy);
+  vec3 vertex = vec3(positions.xy * radiusPixels, 0.);
 
-  vec2 vertex_pos_modelspace[4];
-  vertex_pos_modelspace[0] = projected_coord_xy[0];
-  vertex_pos_modelspace[1] = projected_coord_xy[1];
-  vertex_pos_modelspace[2] = vec2(project_scale(instancePositions.z), 0.0);
-  vertex_pos_modelspace[3] = vec2(1.0, 0.0);
+  gl_Position = project_model_position_and_offset_to_clipspace_fp64(
```

Comment:
I need implement project_model_position_and_offset_to_clipspace_fp64, right?

Reply:
>I need implement project_model_position_and_offset_to_clipspace_fp64, right

The new GLSL function itself is already implemented in this PR: src/core/shaderlib/project64/project64.glsl.js.

* However, the layers need to be adapted to use it. I only updated two layers.

* Also, I think you should carefully consider the name/parameters of this API, I am not convinced that the name I gave it is very good.

* Finally, if we can add a similar function to the 32 bit project module, it could be possible to unify our 32 and 64 bit vertex shaders. That would reduce effort to develop new layers, the cost to maintain old layers and deck.gl code/bundle size.